packages/ui-dom/src/index.js

```diff
@@ -0,0 +1,8 @@
+export { default as Col } from "./col";
+export { default as Row } from "./row";
+export { default as View } from "./view";
+export { default as Button } from "./button";
+export { default as Input } from "./input";
+export { default as RadioButton } from "./radio-button";
+export { default as RadioGroup } from "./radio-group";
+export { default as Text } from "./text";
```

Comment:
I think having an index with all components here is not a good idea for `ui-dom` version. And to keep the same pattern, I wouldn't create an index for native either